randovania/games/fusion/logic_database/header.txt

```diff
@@ -41,6 +41,7 @@ Templates
 * Can Kill Missile Geron:
       Any of the following:
           Any of the following:
+              # TODO: missile gerons actually take only 3 missiles? is this for the s1 cube gerons instead?
```

Comment:
gerons have 30HP
![image](https://github.com/randovania/randovania/assets/125271738/2f3f162b-7811-4c00-a695-5d1e36b93b39)

Reply:
i forgot that this is for theoretical shoot missiles without main missile data. will remove